forge-gui/res/cardsfolder/upcoming/squee_dubious_monarch.txt

```diff
@@ -0,0 +1,10 @@
+Name:Squee, Dubious Monarch
+ManaCost:2 R
+Types:Legendary Creature Goblin Noble
+PT:2/2
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME attacks, create a 1/1 red Golbin creature token that's tapped and attacking.
+SVar:TrigToken:DB$ Token | TokenOwner$ You | TokenAmount$ 1 | TokenScript$ r_1_1_goblin | TokenTapped$ True | TokenAttacking$ True
```

Comment:
`TokenOwner$ You | TokenAmount$ 1` are defaults, not needed – feel free to leave if you like though

Reply:
Removed redundant token params